### PR TITLE
Adding docs for 'pip install mlt' and creating pip installers

### LINKED ISSUE #180: Update README file to mention pip install
Add note on the README about pip installing mlt.  Right now, it just says to use `make`. 

* TBD

### PR DESCRIPTION
**Pull request by ashahba**
fixes #180 

This should only be merged once both #188 is merged and just before building and publishing `mlt` binary. 

Checklist

 - [x] Do you understand it?
 - [x] Is it correct?
 - [x] Is it safe?
 - [x] Is it legally compliant?
 - [x] Is it robust?
 - [x] Is it simple?
 - [x] Is it tested?
 - [x] Is it documented?
 - [x] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +59,14 @@ $ cd mlt
 $ pip install .
 ```
 
+## Create local Python distributions
+```bash
+make dist
+cd dist
+ls mlt*
```
Comment (s1113950) [inline README.md]: Why is there a * here?
Do you mean to show that after the `ls` command you'll see the wheel listed below? Could you do something like 

```
$ ls mlt
mlt-...
```
? I'd put a `$` in front of the commands in the README to distinguish between command and output

File: README.md [hunk 2]
```diff
@@ -61,9 +61,9 @@ $ pip install .
 
 ## Create local Python distributions
 ```bash
-make dist
-cd dist
-ls mlt*
+$ make dist
+$ cd dist
+$ ls mlt*
```
Comment (s1113950) [inline README.md]: why the * here?

Comment (ashahba) [reply README.md]: Because I didn't want to use a specific version here.

**Comment by ashahba** [comment]
@s1113950 I thin it's good for now, but I owe you guys a separate file maybe like `admin_guide.md` under `docs` folder with more documentarians, but that should be a another PR of it's own.

Same thing with testing it.

**Pull request merged**